pages/_app.js

```diff
@@ -18,6 +18,7 @@ import { NextUIProvider, createTheme } from '@nextui-org/react'
 import NavbarComponent from '../components/Navbar/index'
 import { useTranslation } from 'react-i18next'
 import { SSRProvider } from '@react-aria/ssr'
+import BoardBackground from '../components/Background/Board.jsx'
```

Comment:
Should this file be in .jsx? Our current codebase uses .js for all React components. I believe it's better to maintain consistency and use .js for now, even though .jsx could be more appropriate in the long run.

Reply:
@nomadbitcoin , it doesn't necessarily have to be in jsx, I've already changed it to js